scripts/validateSecrets.js

```diff
@@ -0,0 +1,36 @@
+const assert = require('assert');
```

Comment:
I kind of feel like we don't need the validation anymore since a bPanel instance could be connecting to any arbitrary setup and the secrets.env is really just in the case of when you're connecting to a local docker. I'm wondering if there's a good way that we could maybe just bundle the validation and maybe even the creation of the secrets into the docker init process, that way it's much more clearly contained to that process.